content/components.html

```diff
@@ -184,7 +184,7 @@ <h1 class=st-title>Preview article</h1>
             <img
             src=https://neomad.org/static/uploads/avatars/5914dbdca193770016e289a7
             class="avatar avatar-icon"
-            onerror=this.src='./img/avatar.png'
+            onerror=this.src='dist/assets/img/avatar.png'
```

Comment:
`onerror="this.src='dist/assets/img/avatar.png'"`

Quote as there's a spacial char (`=`)